eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
+  PackageName: 'not-specified'
+  ServiceName: 'not-specified'
+  ForRelease: ''
+
+steps:
+  - pwsh: |
+      Import-Module "$(Build.SourcesDirectory)/eng/common/scripts/modules/common-manifest.psd1"
+      $Language = "$(Build.Repository.Name)".Substring("$(Build.Repository.Name)".LastIndexOf('-') + 1)
+      $PackageProp = Get-PkgProperties -PackageName ${{ parameters.PackageName }} -ServiceName ${{ parameters.ServiceName }} -Language $Language -RepoRoot $(Build.SourcesDirectory)
+
+      $ReleaseNotes = (& "$(Build.SourcesDirectory)/eng/common/Extract-ReleaseNotes.ps1" -ChangeLogLocation $PackageProp.pkgChangeLogPath -VersionString $PackageProp.pkgVersion -IncludeTitle $true) -Split [Environment]::NewLine
+
+      # Verify Header Has Unreleased or an has accurate date
+      $ReleaseTitle = $ReleaseNotes[0] -Split ' '
+
+      if ($ReleaseTitle.Length -ne 3)
```

Comment:
This seems like it might be broken. Don't we already have a regex in our Extract-ReleaseNotes that validates the header? If not perhaps we should.

Reply:
This is to ensure the Header also has the release Date or (Unreleased) as the case may be.

Reply:
Lets use the regex to validate this in the common script/module so it is a little more clear what is is doing.